modules/oe_content_news/oe_content_news.post_update.php

```diff
@@ -57,3 +57,28 @@ function oe_content_news_post_update_00004(): void {
     $updated_form_display->save();
   }
 }
+
+/**
+ * Create oe_news_contacts field in the news content type.
+ */
+function oe_content_news_post_update_00005(): void {
+  $storage = new FileStorage(drupal_get_path('module', 'oe_content_news') . '/config/post_updates/00005_create_fields');
```

Comment:
We can move these new fields to oe_content_news_post_update_00003().

Reply:
updated